data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
+homepage: https://www.gov.uk/government/organisations/ministry-of-defence
```

Comment:
We will want to redirect to mainstream content and detailed guides eventually, but we do not have the URLs yet.